docs/api_scrubadub_detectors.rst

```diff
@@ -10,140 +10,237 @@ text.
 Base classes
 ------------
 
-Every ``Detector`` that inherits from ``scrubadub.detectors.base.Detector``.
+Every ``Detector`` that inherits from ``scrubadub.detectors.Detector``.
 
-scrubadub.detectors.base.Detector
----------------------------------
+.. _scrubadub.detectors.Detector:
 
-.. autoclass:: scrubadub.detectors.base.Detector
+scrubadub.detectors.Detector
+----------------------------
 
-scrubadub.detectors.base.RegexDetector
---------------------------------------
+.. autoclass:: scrubadub.detectors.Detector
+
+.. _scrubadub.detectors.RegexDetector:
+
+scrubadub.detectors.RegexDetector
+---------------------------------
 
 For convenience, there is also a ``RegexDetector``, which makes it easy to
 quickly add new types of ``Filth`` that can be identified from regular
 expressions:
 
-.. autoclass:: scrubadub.detectors.base.RegexDetector
+.. autoclass:: scrubadub.detectors.RegexDetector
+
+.. _scrubadub.detectors.RegionLocalisedRegexDetector:
 
-scrubadub.detectors.base.RegionLocalisedRegexDetector
------------------------------------------------------
+scrubadub.detectors.RegionLocalisedRegexDetector
+------------------------------------------------
 
-.. autoclass:: scrubadub.detectors.base.RegionLocalisedRegexDetector
+.. autoclass:: scrubadub.detectors.RegionLocalisedRegexDetector
 
 
 Detectors enabled by default
 ----------------------------
 
 These are the detectors that are enabled in the scrubber by default.
 
+
+.. _scrubadub.detectors.CredentialDetector:
+
 scrubadub.detectors.CredentialDetector
 --------------------------------------
 
 .. autoclass:: scrubadub.detectors.CredentialDetector
```

Comment:
We briefly discussed this - the regex didn't render well, and I am not sure whether we need the actual regex there or not. It may be beneficial but not in its current format?

Reply:
I've removed the value from the webpage.